Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -296,15 +296,13 @@ final class ChangeStreamSpecTests: MongoSwiftTestCase {
     // TODO: SWIFT-560: Run this test.
     func testChangeStreamSpecUnified() throws {
         printSkipMessage(testName: self.name, reason: "Skipping until SWIFT-560 is implemented")
-        return
-
-                // let tests = try retrieveSpecTestFiles(
-                //     specName: "change-streams",
-                //     subdirectory: "unified",
-                //     asType: UnifiedTestFile.self
-                // ).map { $0.1 }
-                // let testRunner = try UnifiedTestRunner()
-                // try testRunner.runFiles(tests)
+        // let tests = try retrieveSpecTestFiles(
```

Comment:
SwiftFormat update starting formatting this a little differently.

coincidentally, we can unskip this pretty soon, when Isabel picks up SWIFT-560.